variables.tf

```diff
@@ -2,8 +2,43 @@
 # Input Variables
 ########################################################################################################################
 
-#variable "my_variable" {
-#  type        = string
-#  description = "A description of my variable"
-#  default     = "default_value"
-#}
+variable "resource_group_id" {
+  description = "The ID of the resource group where the IBM Security Verify instance is created."
+  type        = string
+}
+
+variable "hostname" {
+  type        = string
+  description = "The hostname of the IBM Security Verify instance that is created by this module. Note: After the service instance creation returns, the instance may take up to two minutes to be available."
```

Comment:
Are you adding validation for this? Also mention that its used to generate the dashboard url.